snmp/assets/dashboards/troubleshooting_dashboard.json

```diff
@@ -0,0 +1,2267 @@
+{
+    "author_name": "Datadog",
+    "description": null,
+    "layout_type": "ordered",
+    "template_variables": [
+        {
+            "available_values": [],
+            "default": "*",
+            "name": "Host",
+            "prefix": "host"
+        }
+    ],
+    "title": "Troubleshooting dashboard",
```

Comment:
I think this is too generic. I'm wondering if there is a better name. This dashboard is not only for troubleshooting, but also for visualising the state of the NDM features health.

Thinking about names like:
- NDM Environment
- NDM Setup
- Datadog NDM Environment (Datadog prefix can be useful since unlike other dashboard this one is about Datadog monitoring setup and not customer owned services/devices etc)
- etc

Reply:
Having Datadog in the name makes sense, I think `Datadog NDM Environment` is great